apps/agent/app/src/lib/utils.js

```diff
@@ -1,8 +1,13 @@
 import { round } from './math-utils'
+import { LOCALE_US_FORMAT } from './date-utils'
```

Comment:
This also seems a bit weird to put inside date utils; if we're going export these types of constants, we probably want them in a `locales.js` or etc.